src/main.c

```diff
@@ -48,6 +48,12 @@ static void setup_gpio(void)
 	gpio_set_mode(GPIOC, GPIO_MODE_OUTPUT_50_MHZ,
 		      GPIO_CNF_OUTPUT_PUSHPULL, GPIO13);
 	gpio_clear(GPIOC, GPIO13);
+
+	/* Motor driver */
+	gpio_set_mode(GPIOB, GPIO_MODE_OUTPUT_50_MHZ,
+		      GPIO_CNF_OUTPUT_PUSHPULL,
+		      GPIO12 | GPIO13 | GPIO14 | GPIO15 );
```

Comment:
Remove space

Reply:
Fixed. Added style checks to Travis! :blush: